cmd/oras/internal/option/pusher.go

```diff
@@ -18,26 +18,42 @@ package option
 import (
 	"context"
 	"encoding/json"
+	"errors"
+	"fmt"
 	"os"
+	"regexp"
+	"strings"
 
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/spf13/pflag"
 	"oras.land/oras-go/v2/content"
 )
 
+const (
+	annotationManifest = "$manifest"
+)
+
+var (
+	errAnnotationConflict    = errors.New("annotation input conflict")
```

Comment:
Need more specific error information:

```suggestion
	errAnnotationConflict    = errors.New("annotations cannot be specified via flags and file at the same time")
```

Reply:
updated